packages/appium/docs/en/guides/caps.md

```diff
@@ -67,6 +67,15 @@ Each Appium client has its own way of constructing capabilities and starting a s
 examples of doing this in each client library, head to the [Ecosystem](../ecosystem/index.md) page
 and click through to the appropriate client documentation.
 
+## BiDi
+
+Appium can support [WebDriver BiDi](https://w3c.github.io/webdriver-bidi/) protocol.
+The actual behavior depends on individual drivers.
+
+| Capability Name  | Type     | Description                                    |
+|------------------|----------|------------------------------------------------|
+| `webSocketUrl` | `boolean` | To enable BiDi protocol in the session. |
```

Comment:
it is also important to mention that even if the server itself supports BiDi the particular driver might not, so it is important to check the driver documentation to verify which particular events and commands are supported for which driver versions (if any)

Reply:
Added in the base description section